sdk/java/eclipse-formatter.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
i am not sure how much of the gradle configuration is per-project vs global, but the global configs should be moved into /linters and symlinked (like is done for python, javascript, etc)

Reply:
yea, I was thinking this but forgot to do it 😅

Reply:
is any of the gradle stuff common and can also be moved? (can do this in separate PR)

Reply:
The gradle files are version-specific. If we plan to keep the same version for each project then I think we can move them to somewhere common. 🤔

Reply:
i'd imagine we'd want to use the same gradle version monorepo-wide, or is there some reason we wouldn't?

Reply:
Yeah, it makes sense.  Where would we want to put them?  Jenkins folder?